screens/snackbar/snackbar.js

```diff
@@ -0,0 +1,32 @@
+const btnLeading = document.querySelector("#btn-leading");
+const btnBaseline = document.querySelector("#btn-baseline");
+const closeBtn = document.querySelector("#sb-close-btn");
+const snackbar = document.querySelector("#snackbar");
+const snackbarMsg = document.querySelector("#snackbar-msg");
+
+btnLeading.addEventListener("click", () => {
+  //   snackbar.className = "snackbar sb-leading show-snackbar";
+  if (!snackbar.classList.contains("show-snackbar")) {
```

Comment:
Everything looks good, you can remove these comments if they are not serving any purpose.